ui/StatusQ/src/StatusQ/Components/StatusGroupBox.qml

```diff
@@ -0,0 +1,69 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+import utils 1.0
+
+
+GroupBox {
+    id: root
+
+    topPadding: label.implicitHeight
+    leftPadding: 16
+
+    /*!
+       \qmlproperty string StatusGroupBox::icon
+       This property holds the icon name for the icon represented on top of the
+       component as a title icon.
+    */
+    property url icon
+
+    /*!
+       \qmlproperty int StatusItemSelector::iconSize
+       This property holds the icon size for the icon represented on top of the
+       component as a title icon.
+    */
+    property int iconSize: 18
```

Comment:
That looks like a weird icon size; our icons are usually 20 or 24

Reply:
ok, changed default to 24